docs/datamodel/objects.rst

```diff
@@ -20,7 +20,9 @@ cannot be changed or masked with a different :ref:`computable
 Object types can *extend* other object types, in which case the extending
 type is called a *subtype* and types being extended are called *supertypes*.
 A subtype inherits all links, properties and other aspects of its
-supertypes.
+supertypes. It is possible to extend more than one type - that's called
+*multiple inheritance*. This mechanism allows building complex object types
+out of combinations of more basic types.
```

Comment:
Need to make `extend more than one type` a link to an example or something. The "polymorphic queries" section is one option.